spec/factories/context_notifications.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :context_notification do
+    action { "Published" }
+    association :context, factory: :article
```

Comment:
I can never remember how to do associations in FactoryBot. This is amazing and I will almost certainly not remember it the next time I need it. 😭